### PR TITLE
Solved the task

### PR DESCRIPTION
**Pull request by Likh-Alex**
- Created controller: Implemented creating a driver
- Created controller: Displaying all drivers
- Created controller: Implemented creation of a manufacturer
- Created controller: Implemented creation of a car (with no drivers by default)
- Created controller: Implemented addition of a driver to a car.
**Additionally:**
- Created a controller: Displaying all manufacturers with ability to delete certain manufacturer.
- Implemented an ability to delete certain driver from list of all drivers.
- Created a controller: Displaying all cars (with manufacturer and drivers) with ability to delete certain car.

### CODE REVIEW

**Review by vlykhvar — APPROVED** [review]

File: src/main/java/taxi/controller/AddCarController.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    private static final Injector injector = Injector.getInstance("taxi");
+    private final CarService carService = (CarService) injector
+            .getInstance(CarService.class);
+    private final ManufacturerService manufacturerService = (ManufacturerService) injector
+            .getInstance(ManufacturerService.class);
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp)
+            throws ServletException, IOException {
+        req.getRequestDispatcher("/WEB-INF/car/add.jsp").forward(req,resp);
```
Comment (KseniiaMakarova) [inline src/main/java/taxi/controller/AddCarController.java]:
```suggestion
        req.getRequestDispatcher("/WEB-INF/views/car/add.jsp").forward(req,resp);
```
let's put all the JSPs into `views` folder

File: src/main/java/taxi/controller/IndexController.java [hunk 2]
```diff
@@ -10,6 +10,6 @@
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
-        req.getRequestDispatcher("/WEB-INF/views/index.jsp").forward(req, resp);
+        req.getRequestDispatcher("/WEB-INF/view/index.jsp").forward(req, resp);
```
Comment (KseniiaMakarova) [inline src/main/java/taxi/controller/IndexController.java]:
```suggestion
        req.getRequestDispatcher("/WEB-INF/views/index.jsp").forward(req, resp);
```

File: src/main/webapp/WEB-INF/driver/all.jsp [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+                <c:out value="${driver.name}"/>
+            </td>
+            <td>
+                <c:out value="${driver.licenseNumber}"/>
+            </td>
+            <td><a href="${pageContext.request.contextPath}/drivers/delete?id=${driver.id}">Delete</a></td>
+        </tr>
+    </c:forEach>
+</table>
+<td><a href="${pageContext.request.contextPath}/drivers/add">Add driver</a></td>
```
Comment (KseniiaMakarova) [inline src/main/webapp/WEB-INF/driver/all.jsp]: is it ok that this is outside of the `table` tag?

File: src/main/webapp/WEB-INF/manufacturer/all.jsp [hunk 4]
```diff
@@ -0,0 +1,33 @@
+<%@ taglib prefix="c" uri="http://java.sun.com/jsp/jstl/core" %>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>All drivers</title>
+</head>
+<body>
+<h1>All drivers page</h1>
```
Comment (KseniiaMakarova) [inline src/main/webapp/WEB-INF/manufacturer/all.jsp]:
```suggestion
    <title>All manufacturers</title>
</head>
<body>
<h1>All manufacturers page</h1>
```

**Review by KseniiaMakarova — CHANGES_REQUESTED** [review]
great job!

File: src/main/java/taxi/controller/AddCarController.java (hunk 1 above)
Comment (Likh-Alex) [reply src/main/java/taxi/controller/AddCarController.java]: Done, after all I was a bit confused where they all should be. Thank you for clarification.

File: src/main/java/taxi/controller/IndexController.java (hunk 2 above)
Comment (Likh-Alex) [reply src/main/java/taxi/controller/IndexController.java]: Understood, thank you

File: src/main/webapp/WEB-INF/driver/all.jsp (hunk 3 above)
Comment (Likh-Alex) [reply src/main/webapp/WEB-INF/driver/all.jsp]: Moved it inside the table, table looks better now.Thank you

File: src/main/webapp/WEB-INF/manufacturer/all.jsp (hunk 4 above)
Comment (Likh-Alex) [reply src/main/webapp/WEB-INF/manufacturer/all.jsp]: copy-paste shame) Done

File: src/main/webapp/WEB-INF/views/manufacturer/all.jsp [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+                <c:out value="${manufacturer.name}"/>
+            </td>
+            <td>
+                <c:out value="${manufacturer.country}"/>
+            </td>
+            <td><a href="${pageContext.request.contextPath}/manufacturers/delete?id=${manufacturer.id}">Delete</a></td>
+        </tr>
+    </c:forEach>
+</table>
+<td><a href="${pageContext.request.contextPath}/manufacturers/add">Add manufacturer</a></td>
```
Comment (KseniiaMakarova) [inline src/main/webapp/WEB-INF/views/manufacturer/all.jsp]:
```suggestion
<td><a href="${pageContext.request.contextPath}/manufacturers/add">Add manufacturer</a></td>
</table>
```

**Review by KseniiaMakarova — APPROVED** [review]

File: src/main/webapp/WEB-INF/views/manufacturer/all.jsp (hunk 5 above)
Comment (Likh-Alex) [reply src/main/webapp/WEB-INF/views/manufacturer/all.jsp]: 🤦‍♂️

Comment (Likh-Alex) [reply src/main/webapp/WEB-INF/views/manufacturer/all.jsp]: Fixed

**Pull request merged**